packages/components/psammead-bulletin/src/index.test.jsx

```diff
@@ -10,6 +10,9 @@ const BulletinComponent = ({ script, service, isLive, type, ctaText }) => {
   const headlineText = 'This is the headline';
   const ctaLink = 'https://bbc.co.uk';
 
+  const liveCtaText = isLive ? `${ctaText} Live` : ctaText;
```

Comment:
I would leave this as a `ctaText` because not always is gonna be live.

Reply:
how about `playCtaText`?

Reply:
SGTM